app/services/onboarding.js

```diff
@@ -117,21 +117,23 @@ export default class OnboardingService extends Service {
 
   async getApplicationDetails() {
     try {
-      const userId = this.login.userData.id;
-      const applicationResponse = await fetch(
-        `${APPS.API_BACKEND}/applications?userId=${userId}`,
-        {
-          credentials: 'include',
-        },
-      );
-      const applicationData = await applicationResponse.json();
-
-      this.applicationData = applicationData?.applications?.[0];
-
-      this.loadingApplicationData = false;
+      const userId = this.login.userData?.id;
+      if (userId) {
+        const applicationResponse = await fetch(
+          `${APPS.API_BACKEND}/applications?userId=${userId}`,
+          {
+            credentials: 'include',
+          },
+        );
```

Comment:
If we have Adapters, models and serializers setup in this repo can we please make this fetch call using it?

If not then all good

Reply:
For join we are not using adapter